src/lib/route/utils/route-children.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { Route } from '@angular/router';
+import { getChildren } from './route-children';
+
+describe('getChild-function', () => {
+    it('should return the children to a route', () => {
+        const route: Route = {
+            path: 'foo',
+            children: [{ path: 'bar', data: { label: 'bar' } }]
+        };
+        expect(getChildren(route)).toEqual([{ path: 'bar', data: { label: 'bar' } }]);
+    });
+
+    it('should return undefined with empty children', () => {
+        const route: Route = {
+            path: 'foo'
+        };
+
+        expect(getChildren(route)).toBeUndefined();
+    });
+
+    it('should return lazy-loaded routes', () => {
+        const route: Route = {
+            path: 'foo'
+        };
+
+        route['_loadedConfig'] = { routes: [{ path: 'bar', data: { label: 'bar' } }] };
```

Comment:
```suggestion
        const route: Route = {
            path: 'foo',
            '_loadedConfig': { routes: [{ path: 'bar', data: { label: 'bar' } }] }
        };
```
Oder meckert typescript dann?

Reply:
Typescript meckert dann

Reply:
k 👌